src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -2006,6 +2060,14 @@ def CreateDropDown(width: int, items: list[str], selectedIndex: int = 0) -> PyCo
     """
     pass
 
+def CreateModernGump(x: int, y: int, width: int, height: int, resizable: bool = True, minWidth: int = 50, minHeight: int = 50, onResized: Any = None) -> Any:
```

Ruff (0.13.1)

Undefined name `Any`

(F821)